main.js

```diff
@@ -1,36 +1,38 @@
 /* global __dirname */
 /* global process */
-const electron = require('electron')
+const electron = require("electron")
```

Comment:
Why change all the quotes from `'` to `"`?

Reply:
i didn't.. i JUST forked and cloned... starting from scratch and edited the few lines, and pushed it back...

Reply:
oh, i ran eslint to fix the tabs..  I bet it changed the quotes...